r-package/grf/R/forest_summary.R

```diff
@@ -28,7 +28,9 @@
 #'
 #' @export
 test_calibration = function(forest) {
-  
+
+  forest$Y.orig <- unname(forest$Y.orig) # remove column name if any
```

Comment:
That's a good point that I didn't consider. I've moved the `unname()` function so that it only applies when making either `DF`. I'll update the pull request. 

And thanks for the continued development on this great package!